qa/tasks/ceph.py

```diff
@@ -1926,6 +2104,33 @@ def task(ctx, config):
             cluster=config['cluster'],
         )
 
+        if ctx.ceph_volume:
```

Comment:
It would help future readers to add a few comments about what this is doing and why - perhaps in a docstring for _fix_roles_map()